setup.py

```diff
@@ -2,28 +2,27 @@
 
 
 def readme():
-    with open('README.md') as f:
+    with open("README.md") as f:
         return f.read()
 
+
+_EXTRAS = {"yasg": ["drf-yasg==1.12.1"], "filters": ["django-filter>=1.1.0"]}
+_EXTRAS["all"] = sorted([req for req_list in _EXTRAS.values() for req in req_list])
+
 setup(
-    name='drf-jsonapi',
+    name="drf-jsonapi",
     use_scm_version=True,
-    license='MIT',
-    description='OpenSource Django JSONAPI Library',
+    license="MIT",
+    description="OpenSource Django JSONAPI Library",
     long_description=readme(),
-    keywords='django jsonapi',
-    author='Vacasa, LLC',
-    author_email='opensource@vacasa.com',
-    url='https://github.com/vacasa/drf-jsonapi',
-    packages=find_packages(exclude=['tests*']),
+    keywords="django jsonapi",
+    author="Vacasa, LLC",
+    author_email="opensource@vacasa.com",
+    url="https://github.com/vacasa/drf-jsonapi",
+    packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    install_requires=[
-        'django',
-        'djangorestframework',
-        'drf-nested-routers',
-        'django-filter',
-        'drf-yasg',
-    ],
+    install_requires=["Django>=2.0.6", "djangorestframework>=3.8.2"],
```

Comment:
Install should require `drf_yasg` 1.12 or greater.

Reply:
Is it actually required or an extra? Because right now it's defined up above as an extra, installable as `drf-jsonapi[yasg]`.